scripts/main/album.js

```diff
@@ -20,7 +20,7 @@ album.isSmartID = function (id) {
  * @returns {boolean}
  */
 album.isSearchID = function (id) {
-	return id === SearchAlbumIDPrefix || id.startsWith(SearchAlbumIDPrefix + "/");
+	return id !== null && id.startsWith(SearchAlbumIDPrefix);
```

Comment:
```suggestion
	return id !== null && (id === SearchAlbumIDPrefix || id.startsWith(SearchAlbumIDPrefix + "/"));
```

There's a reason why I coded it that way in the first place :smiley:. Imagine a regular album ID that happens to start with the string `search` -- not very probable, but far from impossible...

Reply:
Okay ..... I actually assumed so, but than considered that we have 64 equally random symbols (a-z, A-Z, 0-9, -, _) and "search" has 6 characters, hence the chance is 1/64^6 ≈ 1/10^11, i.e. one time in more than a billion.

But, yes, better safe than sorry.